packages/scripts.nix

```diff
@@ -265,7 +265,7 @@
           ${pkgs.kata.genpolicy}/bin/genpolicy < "$tmpdir/coordinator_base.yml"
         ;;
         *)
-          echo "Unsupported platform: {{ platform }}" >&2
+          echo "Unsupported platform: $platform" >&2
```

Comment:
```suggestion
          echo "Unsupported platform: ${platform}" >&2
```
nit

Reply:
That's not working here as this is an inline string in nix.